test/core/compression/BUILD

```diff
@@ -12,23 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("//bazel:grpc_build_system.bzl", "grpc_cc_test", "grpc_package")
+load(
+    "//bazel:grpc_build_system.bzl",
+    "grpc_cc_library",
+    "grpc_cc_test",
+    "grpc_package",
+)
 load("//test/core/util:grpc_fuzzer.bzl", "grpc_fuzzer")
 
 grpc_package(name = "test/core/compression")
 
 licenses(["notice"])
 
-grpc_cc_test(
-    name = "algorithm_test",
-    srcs = ["algorithm_test.cc"],
-    language = "C++",
-    uses_polling = False,
-    deps = [
-        "//:gpr",
-        "//:grpc",
-        "//test/core/util:grpc_test_util",
-    ],
+grpc_cc_library(
+    name = "args_utils",
+    testonly = 1,
+    srcs = ["args_utils.cc"],
+    hdrs = ["args_utils.h"],
+    visibility = ["//visibility:public"],
```

Comment:
Why public?  Is it used outside of this directory?

Reply:
It's used in the end2end tests also